src/components/Molecules/SearchInput/SearchInput.md

```diff
@@ -0,0 +1,15 @@
+# Search input
+
+initialState = { search: '' };
+
+```js
+<SearchInput
```

Comment:
Is it possible to include loading ui here as well, it can use isLoading bool prop to be displayed,
currently it appears like that : https://drive.google.com/file/d/1znOFE3W0s9zhN2oEMJwqZYrhSsPzpg6A/view?usp=sharing

Reply:
Yeah, I think we need to create a separate ticket for that.

Reply:
ok